### ISSUE TITLE
Fix concurent modification exception

### ISSUE THREAD

**Issue by RkShaRkz** [RkShaRkz/Starsector-IntelliJ-Template#19]
fix this
```
4998314 [Thread-2] ERROR com.fs.starfarer.combat.CombatMain  - java.util.ConcurrentModificationException
java.util.ConcurrentModificationException: null
        at java.base/java.util.ArrayList$Itr.checkForComodification(ArrayList.java:1096) ~[?:?]
        at java.base/java.util.ArrayList$Itr.next(ArrayList.java:1050) ~[?:?]
        at data.scripts.campaign.intel.VayraPersonBountyManager.checkForFuckedUpParticipants(VayraPersonBountyManager.java:76) ~[?:?]
        at data.scripts.campaign.intel.VayraPersonBountyManager.advance(VayraPersonBountyManager.java:268) ~[?:?]
        at data.scripts.VayraMergedModPlugin.onNewGameAfterTimePass(VayraMergedModPlugin.java:343) ~[?:?]
        at com.fs.starfarer.campaign.save.CampaignGameManager.o00000(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.title.TitleScreenState.dialogDismissed(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.O.dismiss(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.impl.for.dismiss(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.campaign.save.if.actionPerformed(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.o0O0.buttonPressed(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.int.├Æ00000(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.int.processInput(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.W.o00000(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.BaseGameState.traverse(Unknown Source) ~[port_obf.jar:?]
        at com.fs.state.AppDriver.begin(Unknown Source) ~[port.common_obf.jar:?]
        at com.fs.starfarer.combat.CombatMain.main(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.StarfarerLauncher.super(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.StarfarerLauncher$1.run(Unknown Source) ~[port_obf.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1575) [?:?]
```

**Comment by RkShaRkz** [comment]
taken care of by PR https://github.com/RkShaRkz/Starsector-IntelliJ-Template/pull/21 (commit 47a1ca5)

Will close when PR gets merged in. Will hold off releasing a new version since i want to take a good jab at the vanishing planet conditions too. Ping me on discord for hotfix if you're blocked, it's been confirmed as resolving the issue.

**Comment by RkShaRkz** [comment]
hofix uploaded to issue #18

**Comment by RkShaRkz** [comment]
Issue was resolved by 8bc9dbe - closing until it resurfaces on new 3.3.8 version.
